src/azure-cli/azure/cli/command_modules/servicebus/custom.py

```diff
@@ -673,6 +675,7 @@ def return_valid_duration_create(update_value):
     from datetime import timedelta
     from isodate import parse_duration
     from knack.util import CLIError
+    import warnings
```

Comment:
It is not Azure CLI convention to use `warnings` module. Instead, warnings in Azure CLI is written like

https://github.com/Azure/azure-cli/blob/d331edf76e5866cda1e6e9641d935256fc4f03ff/src/azure-cli/azure/cli/command_modules/profile/custom.py#L38-L39